LocalSettings.php

```diff
@@ -6450,6 +6450,7 @@
 		'nenawikiwiki' => 'America/New_York',
 		'ontariobrasswiki' => 'America/Toronto',
 		'proxybotwiki' => 'America/Chicago',
+		'revitwiki' => 'Pacific Ocean/Honolulu',
```

Comment:
It should be just "Pacific" not "Pacific Ocean"

Reply:
@Reception123 If I go into my time zone preferences on-wiki I see it listed as Pacific Ocean/Honolulu.